src/app.ts

```diff
@@ -1,8 +1,11 @@
 import express from 'express'
-
+import path from 'path'
 const app = express()
 const port = 3000
 
-app.get('/', (req, res) => res.send('Hello World!'))
+app.set('view engine', 'ejs')
+app.set('views', path.join(__dirname, '../../views'))
```

Comment:
could we have `views` inside the `src` folder please? I think its better to keep the root folder clear.